packages/forms/src/elements/helpers.ts

```diff
@@ -89,6 +89,23 @@ export const formatWithDecimals = (value: string, decimals: number): string => {
 	return intValueFormatted + decimalPart;
 };
 
+export const getFinalValueWithFormat = (
+	value: string,
+	decimals: number,
+): string => {
+	const newValueWithNoCommas = value.replace(/,/g, '');
```

Comment:
Will value here ever be null or undefined?

Reply:
it shouldn't, because the only place where is called is in `valueLengthValid`, which already checks `if(value)`, before doing any operation.